packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -317,14 +319,16 @@ const _BaseButton: React.ForwardRefRenderFunction<BladeElementRef, BaseButtonPro
     onKeyDown,
     type = 'button',
     children,
-    accessibilityLabel,
     testID,
     onFocus,
     onMouseLeave,
     onMouseMove,
     onPointerDown,
     onPointerEnter,
-    ...styledProps
+    accessibilityProps,
+    onTouchEnd,
+    onTouchStart,
```

Comment:
new props?

Reply:
They were part of Common events type but were not explicitly added in basebutton and baselink. They were still getting passed and we were using them so added them explicitly